lib/routes/i-cable/news.ts

```diff
@@ -0,0 +1,73 @@
+import { Route } from '@/types';
+import cache from '@/utils/cache';
+import got from '@/utils/got';
+import { getCurrentPath } from '@/utils/helpers';
+import path from 'node:path';
+import { art } from '@/utils/render';
+import { config } from '@/config';
+
+const __dirname = getCurrentPath(import.meta.url);
+
+export const route: Route = {
+    path: '/news/:category?',
+    categories: ['traditional-media'],
+    example: '/i-cable/news',
+    parameters: { category: '分類，默認為新聞資訊' },
+    features: {
+        requireConfig: false,
+        requirePuppeteer: false,
+        antiCrawler: false,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    radar: [
+        {
+            source: ['www.i-cable.com'],
+            target: '/news',
+        },
+        {
+            source: ['www.i-cable.com/category/:category'],
+            target: '/news/:category',
+        },
+    ],
+    name: '新聞',
+    maintainers: ['quiniapiezoelectricity'],
+    handler,
+    url: 'www.i-cable.com/',
+    description: `
+:::tip
+分類只可用分類名稱，如：新聞資訊/港聞
+:::`,
+};
+
+async function handler(ctx) {
+    const category = ctx.req.param('category') ?? '新聞資訊';
+    const limit = ctx.req.query('limit') ?? 20;
+    const root = 'https://www.i-cable.com/wp-json/wp/v2';
+
+    const response = await cache.tryGet(`${root}/categories?slug=${category}`, async () => await got(`${root}/categories?slug=${category}`), config.cache.routeExpire, false);
+    const metadata = response.data[0];
```

Comment:
`.data[0]` maybe empty if user provides an invalid input so please throw an exception if that happens